src/flashbar/styles.scss

```diff
@@ -80,7 +80,7 @@
 }
 
 .flash-text {
-  margin-block: awsui.$border-field-width;
+  margin-block: awsui.$border-width-field;
```

Comment:
similar: I don't see any reason why this should semantically be linked to form field border width (also affects the related `calc` in the file above)

Reply:
![Screenshot 2024-11-19 at 09 22 02](https://github.com/user-attachments/assets/3e73ff3d-defc-4360-8b9c-f57bc395dc48)

Reply:
Per offline discussion leaving this as is.